packages/typescript-estree/tests/lib/convert.test.ts

```diff
@@ -21,73 +21,77 @@ describe('convert', () => {
     );
   }
 
-  it('deeplyCopy should convert node correctly', () => {
-    const ast = convertCode('type foo = ?foo<T> | ?(() => void)?');
+  /* eslint-disable @typescript-eslint/dot-notation */
```

Comment:
why is this disabled?

Reply:
`deeplyCopy` is a private member. `['deeplyCopy']` bypasses that.